google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -1919,6 +1926,12 @@ def insert rows, skip_invalid: nil, ignore_unknown: nil
         # @param [Boolean] ignore_unknown Accept rows that contain values that
         #   do not match the schema. The unknown values are ignored. Default is
         #   false, which treats unknown values as errors.
+        # @param [Array<String>] insert_ids A unique ID for each row. BigQuery
```

Comment:
I am confused why `insert_ids` is an argument to create the inserter object. I think it should only be present on `AsyncInserter#insert`.

Reply:
Yes, see reply above.